app/views/posts/show.html.erb

```diff
@@ -14,4 +14,14 @@
       <p><%=comment.author.name%>: <%=comment.text%></p>
     <%end%>
   </div>
+  <div class='button-see-all-posts'>
+    <%=link_to new_comment_path do%>
+    <button>Create New Comment</button>
+  <%end%>
+  </div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make when a `User` creates a `Comment` it also updates the `comments_counter`. Right now, after creating `Comments` they are displayed on the **UI** but the `Comments_counter` still have `0` as value but when checking in the `database` you notice that they were created successfully.
![Screenshot from 2023-02-02 07-13-00](https://user-images.githubusercontent.com/83042742/216237580-44c33012-86c1-4dc0-8490-6f6e903da3dd.png)


## NOTICE: 
I really recommend you fix all the above changes before merging your pull request even it they are **OPTIONAL**